.github/workflows/python_testing.yml

```diff
@@ -0,0 +1,31 @@
+name: python testing on push and PR
+#
+# Run the python directory testing on push and pull request
+#
+on: [push, pull_request]
+
+jobs:
+  pytest:
+     runs-on: ubuntu-latest
+     steps:
+        # Checkout the code
+        - uses: actions/checkout@v2
+        - uses: conda-incubator/setup-miniconda@v2
+          with:
+            activate-environment: ctsm_pylib
+            environment-file: python/conda_env_ctsm_py.txt
+            auto-activate-base: false
+            channels: conda-forge
+        - name: manage_exernals and LFS
+          run: |
+            git lfs install
+            manage_externals/checkout_externals
+        - name: chdirectory python
+          run: |
+              cd python
+        - name: Run python tests
+          run: |
+              make test
+        - name: Run pylint
+          run: |
+              make lint
```

Comment:
@slevisconsulting this is a new action for running python testing on pushes and PR's. It looks like it's failing right now, so it needs more work. Is this something you'd like to dig into to get working? Maybe we should meet to discuss it next week....

Reply:
This is working on #1580

Reply:
@ekluzek and I agreed that this is not within the scope of this PR, so we're leaving unresolved.

Reply:
Remove python_testing.yml from this PR